test/pkg/fixture/fixture.go

```diff
@@ -199,7 +199,7 @@ func NewStressTestDeployment(name, namespace string, extraLabels map[string]stri
 								},
 								Limits: corev1.ResourceList{
 									corev1.ResourceCPU:    resource.MustParse("1"),
-									corev1.ResourceMemory: resource.MustParse("100M"),
+									corev1.ResourceMemory: resource.MustParse("150M"),
```

Comment:
Why do we need to increase that limit?

Reply:
There are occations that the pod with stress chaos would use more than 100M memory and gets OOMed. I'm not sure why this would happen.